woocommerce/wc-admin
# Add icon to snackbar

_Note that this is WIP, waiting on a final SVG for the pencil icon (@elizaan36)._

Fixes #5436 

This adds an optional icon to the snackbar, and uses the icon when the snackbar is displayed for CES.
![image](https://user-images.githubusercontent.com/224531/98758492-afac8200-241a-11eb-8eb3-725c20402146.png)

*Testing instructions*

- `DELETE FROM wp_options WHERE option_name = 'woocommerce_ces_shown_for_actions'`
- Open a product for editing
- Edit the product and click Update
- Wait for a while and the snackbar should appear
- Make sure the snackbar has the icon and is aligned correctly
